SignalR.Client/SRConnectionInterface.h

```diff
@@ -49,6 +49,12 @@
 @property (strong, nonatomic, readwrite) NSMutableDictionary *headers;
 
 ///-------------------------------
+/// @name Internal state
+///-------------------------------
+
+- (NSString *)onSending;//TODO: this just encapsulates connectionData. can we pull this into a getUrl like js client does?
```

Comment:
We can remove this from the interface bc we were able to get control flow working nicely inside stop